content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,278 @@
+## Overview
+Datadog Synthetics offers two different ways to monitor your applications: API tests to monitor the uptime of your API endpoints and browser tests to check key user journeys. Your tests can be run either from managed locations or from private locations. Synthetics helps you ensure uptime, identify regional issues, and make sure key web transactions can be performed on your application.
+
+{{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" responsive="true">}}
+
+By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing, as experienced by your users. The [Synthetics][1] homepage details all of this information to give you real-time status updates, response times, and uptimes.
+
+This guide shows you how to set up your first Synthetics test with Datadog. Follow the sections below to configure an API test, browser test, or private location for internal-facing apps or private URLs.
+
+## Prerequisites
```

Comment:
It might make more sense to split prerequisites by getting started section.
For instance the extension is only required to run browser tests, not for API tests. Private locations also have some specific requirements, such as having a host that can run Docker, and Linux containers.